cves/CVE-2017-7679.yml

```diff
@@ -152,10 +193,16 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: |
+    The vulnerability was introduced 19 years ago and expanded the capabilities
+    of the mod_mime.c file. It seems that content types are being dynamically added
+    to some sort of server side pool, but I'm not 100% sure about that. 
+
+    The code has had regular additions/modifications over the course of the last
+    19 years, so this isn't a file that was just created and left alone "forever".
```

Comment:
Were you able to tell if any of the modifications changed the vulnerable code at all, or were they in unrelated areas/just style changes?

Reply:
From a brief glance at the string of commits, it seems that the modifications were primarily just style/context changes. The lines causing the vulnerability appear to be the same.